stories/Dropdown.stories.tsx

```diff
@@ -245,6 +246,16 @@ export const Select: React.FC = () => (
   </div>
 );
 
+export const Portal = PortalTemplate.bind({});
+Portal.args = {
+  title: 'A Dropdown Menu in a Portal',
+  component: <Dropdown
+    options={defaultOptions}
+    menuPortalTarget={document.body}
+    menuShouldBlockScroll={true}
+    menuPortalStyles={{ zIndex: 100 }} />,
```

Comment:
beside the zIndex, is there another use?

Reply:
is it really necessary to adjust the zIndex or is it just for demo purpose?